yarn.lock

```diff
@@ -792,23 +792,6 @@
     path-normalize "^6.0.13"
     zod "^3.22.4"
 
-"@tact-lang/compiler@~1.6.1":
```

Comment:
It seems correct to me; the recent version of Misti requires Tact 1.6.2 which is specified above.